vividus-soft-assert/src/main/java/org/vividus/softassert/formatter/AssertionFormatter.java

```diff
@@ -38,21 +43,16 @@ public String getMessage(String description, KnownIssue issue)
         StringBuilder message = new StringBuilder(issue.isPotentiallyKnown() ? "Potentially known" : "Known")
                 .append(" issue: ")
                 .append(issue.getIdentifier())
-                .append(" (Type: ")
-                .append(issue.getType())
-                .append(DOT_CHAR);
-        String status = issue.getStatus();
-        if (status != null)
-        {
-            message.append(" Status: ").append(status);
-        }
-        String resolution = issue.getResolution();
-        if (resolution != null)
-        {
-            message.append(DOT_CHAR).append(" Resolution: ").append(resolution);
-        }
-        message.append("). ").append(description);
-        return message.toString();
+                .append(" (");
+
+        Map<String, Object> details = new LinkedHashMap<>();
+        details.put("Type", issue.getType());
```

Comment:
do you really need one more map? why not to add these values to string builder explicitly?